ThatValidator.js

```diff
@@ -362,10 +347,10 @@
 
                 for(var i = 0; i < elements.length; i++)
                 {
-                    var field = elements[i];
+                    var field2 = elements[i];
 
                     //if the element has not already been found
-                    if(self.fields.indexOf(field) == -1)
+                    if(self.fields.indexOf(field2) == -1)
```

Comment:
Not sure about this. Where is the conflict? Where is `field` even defined now to add the handlers?